runtime/src/inline_feature_gate_program.rs

```diff
@@ -0,0 +1,5 @@
+//! Contains replacement program IDs for the feature gate program
+
+pub(crate) mod noop_program {
+    solana_sdk::declare_id!("5Re2FUHmvSdem1KV2bU5GFvqZpP2MXj76yVhMKKbP45o");
```

Comment:
Just putting a reminder here for myself once the verifiable build stuff is done. Replace with new program ID